todo_app/app.py

```diff
@@ -11,132 +11,184 @@
-            dbconnect, ssl=True, ssl_cert_reqs='CERT_NONE')
+            connectString, ssl=True, ssl_cert_reqs='CERT_NONE')
         database = client["01"]
         date_now = datetime.now().strftime('%Y-%m-%d')
 
-        post = {"name": name, "idBoard": "6005828032dafa5707bf5dc3",
+        post = {"name": name, "idBoard": todoBoard,
                 "last_modified": date_now}
 
         trello_collection = database["trello_collection"]
         result = trello_collection.insert_one(post)
+        app.logger.info("Result is %s", result)
 
     def update_item(id):
         client = pymongo.MongoClient(
-            dbconnect, ssl=True, ssl_cert_reqs='CERT_NONE')
+            connectString, ssl=True, ssl_cert_reqs='CERT_NONE')
         database = client["01"]
         post = {"_id": ObjectId(id)}
         trello_collection = database["trello_collection"]
         date_now = datetime.now().strftime('%Y-%m-%d')
         result = trello_collection.update_one(
-            post, {"$set": {"idBoard": "6005828032dafa5707bf5dc5",
+            post, {"$set": {"idBoard": doneBoard,
                             "last_modified": date_now}}
+
         )
+        app.logger.info(" Updated Result is %s", result)
 
     def return_todo(id):
         client = pymongo.MongoClient(
-            dbconnect, ssl=True, ssl_cert_reqs='CERT_NONE')
+            connectString, ssl=True, ssl_cert_reqs='CERT_NONE')
         database = client["01"]
         post = {"_id": ObjectId(id)}
         trello_collection = database["trello_collection"]
         date_now = datetime.now().strftime('%Y-%m-%d')
         result = trello_collection.update_one(
-            post, {"$set": {"idBoard": "6005828032dafa5707bf5dc3",
+            post, {"$set": {"idBoard": todoBoard,
                             "last_modified": date_now}}
+
         )
+        app.logger.info("Result is %s", result)
 
-    # def show_done():
-    #    client = pymongo.MongoClient(
-    #        "mongodb+srv://admin:MongoAdmin1@cluster0.qtpde.mongodb.net/myFirstDatabase?retryWrites=true&w=majority", ssl=True, ssl_cert_reqs='CERT_NONE')
-    #    database = client["01"]
-    #    trello_collection = database["trello_collection"]
-    #    date_now = datetime.now().strftime('%Y-%m-%d')
-    #    result = trello_collection.find(
-    #        {"idBoard": "6005828032dafa5707bf5dc5",
-    #         "last_modified": date_now}
-    #    )
-
-    @app.route('/')
+    @ app.route('/')
+    @ login_required
     def index():
         connectDb()
-        # show_done()
         client = pymongo.MongoClient(
-            dbconnect, ssl=True, ssl_cert_reqs='CERT_NONE')
+            connectString, ssl=True, ssl_cert_reqs='CERT_NONE')
         database = client["01"]
         trello_collection = database["trello_collection"]
         todo = trello_collection.find(
-            {'idBoard': '6005828032dafa5707bf5dc3'}, {"name": 1})
+            {'idBoard': todoBoard}, {"name": 1})
+        app.logger.info("Todo is %s", todo)
```

Comment:
This prints something like `INFO in app: Todo is <pymongo.cursor.Cursor object at 0x00000151B71DF940>`, which isn't that useful